regression/cpp/Makefile

```diff
@@ -10,7 +10,7 @@ else
 endif
 
 ifeq ($(BUILD_ENV_),MSVC)
-	excluded_tests = -X gcc-only
+	excluded_tests = -X gcc-only -X winbug
```

Comment:
I am kinda surprised this wasn't failing on the old codebuild job. Obviously not a blocking comment, but I wonder why that was passing.

Reply:
I have had a thought about this, and I've arrived at two theories regarding this, but I can't test them now:

1) The `makefiles` were misconfigured before, not including whole folders from `regression/` so it might just be that some tests were skipped by accident.
2) Even though the `_BUILD_ENV` was also specified to be `MSVC`, it ended up being overridden to `Cygwin` because of the build environment, which was calling `g++` which does not throw compiler errors for these tests (`cl.exe` does though, which is what `clcache` is invoking).

Reply:
Hmm. I thought this was supposed to be a visual studio build. I.E. not using g++ at all. Still not a blocking comment though, as we need to get this merged rather than continuing to have no tests of Windows + make.

Reply:
This *is* a Visual Studio build only - I've gone to great lengths to ensure that the only compiler used is `cl.exe` through `clcache`.

I'm talking about the previous code build job this is replicating. The way I said I'm assuming it ended up using `g++` is a weird explicit setup of the path to Cygwin tools before it called `make`

Reply:
Ah, now I understand. :+1: It makes sense that the tests could have been failing silently before due to incorrect setup.